--- etc/init.d/ssh.orig	2015-09-05 11:06:18.070820595 +0200
+++ etc/init.d/ssh	2015-09-05 11:10:58.398820595 +0200
@@ -20,10 +20,20 @@
 
 if test -f /etc/default/ssh; then
     . /etc/default/ssh
 fi
 
+if [ ! -e /etc/ssh/ssh_host_rsa_key ]; then
+    echo "Generating SSH host keys..." || true
+    mount -n -o remount,rw / || echo "Remounting / rw failed"
+    ssh-keygen -q -t rsa -N "" -f /etc/ssh/ssh_host_rsa_key
+    ssh-keygen -q -t dsa -N "" -f /etc/ssh/ssh_host_dsa_key
+    ssh-keygen -q -t ecdsa -N "" -f /etc/ssh/ssh_host_ecdsa_key
+    sync || echo "sync failed"
+    mount -n -o remount,ro / || echo "Remounting / ro failed"
+fi
+
 . /lib/lsb/init-functions
 
 if [ -n "$2" ]; then
     SSHD_OPTS="$SSHD_OPTS $2"
 fi
