--- etc/init.d/ssh.org  2015-09-05 14:14:02.210820595 +0200
+++ etc/init.d/ssh      2015-09-05 14:14:57.438820595 +0200
@@ -20,10 +20,12 @@

 if test -f /etc/default/ssh; then
     . /etc/default/ssh
 fi

+/etc/createHostKey.sh
+
 . /lib/lsb/init-functions

 if [ -n "$2" ]; then
     SSHD_OPTS="$SSHD_OPTS $2"
 fi
--- etc/systemd/system/sshd.service.org	2015-09-05 13:46:43.218820595 +0200
+++ etc/systemd/system/sshd.service	2015-09-05 13:46:54.302820595 +0200
@@ -3,10 +3,11 @@
 After=network.target auditd.service
 ConditionPathExists=!/etc/ssh/sshd_not_to_be_run
 
 [Service]
 EnvironmentFile=-/etc/default/ssh
+ExecStartPre=/etc/createHostKey.sh
 ExecStart=/usr/sbin/sshd -D $SSHD_OPTS
 ExecReload=/bin/kill -HUP $MAINPID
 KillMode=process
 Restart=on-failure
--- lib/systemd/system/ssh.service.org	2015-09-05 15:31:59.666820595 +0200
+++ lib/systemd/system/ssh.service	2015-09-05 15:32:25.262820595 +0200
@@ -3,10 +3,11 @@
 After=network.target auditd.service
 ConditionPathExists=!/etc/ssh/sshd_not_to_be_run
 
 [Service]
 EnvironmentFile=-/etc/default/ssh
+ExecStartPre=/etc/createHostKey.sh
 ExecStart=/usr/sbin/sshd -D $SSHD_OPTS
 ExecReload=/bin/kill -HUP $MAINPID
 KillMode=process
 Restart=on-failure
 
