--- etc/systemd/system/sshd.service.org	2015-09-05 13:46:43.218820595 +0200
+++ etc/systemd/system/sshd.service	2015-09-05 13:46:54.302820595 +0200
@@ -3,10 +3,11 @@
 After=network.target auditd.service
 ConditionPathExists=!/etc/ssh/sshd_not_to_be_run
 
 [Service]
 EnvironmentFile=-/etc/default/ssh
+ExecStartPre=/etc/createHostKey.sh
 ExecStart=/usr/sbin/sshd -D $SSHD_OPTS
 ExecReload=/bin/kill -HUP $MAINPID
 KillMode=process
 Restart=on-failure
 
