--- etc/init.d/udev.orig	2015-09-04 18:30:50.705512432 +0200
+++ etc/init.d/udev	2015-09-04 18:31:46.817512432 +0200
@@ -161,10 +161,13 @@
 case "$1" in
     start)
     if init_is_upstart 2>/dev/null; then
 	exit 1
     fi
+    if [ ! -d /tmp/udev-rules.d ]; then
+       mkdir -p /tmp/udev-rules.d
+    fi
     if mountpoint -q /dev/; then
 	TMPFS_MOUNTED=1
     elif [ -e "/dev/.udev/" ]; then
 	log_warning_msg ".udev/ already exists on the static /dev"
     fi
